Documentation/contributing/release/feature.rst

```diff
@@ -48,15 +56,45 @@ On Freeze date
        git commit
        git push
 
-#. Follow the :ref:`release_candidate_process` to release ``v1.2.0-rc1``.
-
 #. Create the following GitHub labels:
 
    #. ``backport-pending/1.2``
    #. ``backport-done/1.2``
    #. ``backport/1.2``
    #. ``needs-backport/1.2``
 
+
+#. Checkout to master and update the ``.github/cilium-actions.yml`` to have
+   all the necessary configurations for the backport of the new ``vX.Y`` branch.
+
+   ::
+
+       git checkout master
+       git checkout -b pr/master-cilium-actions-update
```

Comment:
nit: The above two could be combined into `git checkout -b pr/master-cilium-actions-update master`

(The last `master` could even be `origin/master`)